src/slick.grid.ts

```diff
@@ -3899,25 +3924,50 @@ export class SlickGrid<TData = any, C extends Column<TData> = Column<TData>, O e
     if ((formatterResult as FormatterResultObject)?.addClasses) {
       addlCssClasses += (addlCssClasses ? ' ' : '') + (formatterResult as FormatterResultObject).addClasses;
     }
-    const toolTip = (formatterResult as FormatterResultObject)?.toolTip ? `title="${(formatterResult as FormatterResultObject).toolTip}"` : '';
 
-    let customAttrStr = '';
-    if (m.hasOwnProperty('cellAttrs') && m.cellAttrs instanceof Object) {
-      for (const key in m.cellAttrs) {
-        if (m.cellAttrs.hasOwnProperty(key)) {
-          customAttrStr += ` ${key}="${m.cellAttrs[key]}" `;
+    if (this._options.nonce) {
```

Comment:
same comment as before, I don't think I want to keep the previous code with html string, just keep the pure HTML part unless the perf has a big hit. I was ok to split the code in DataView for other reasons but doubling the code here is just too much trouble to maintain and don't want the size of the lib to to get bigger either

Reply:
okay, I'll check how big of a difference it is on performance, then before removing the old code

Reply:
Performance wise
[with nonce.json](https://github.com/6pac/SlickGrid/files/13305991/with.nonce.json)
[without nonce.json](https://github.com/6pac/SlickGrid/files/13305992/without.nonce.json)
here a lot of numbers, would tho say it seems to be in margin of error, tho specific cases it can be up to double as slow

this is just the tests run once of the renderRows function through the cypress test